File: mobile/src/main/javaBackup/com/planit/home/planit007/data/Contracts/GroupContract.java
package planit007.planit.home.planit007.data.Contracts;

import android.provider.BaseColumns;

/**
 * Created by Home on 2017-08-02.
 */

public class GroupContract {

    public class GroupEntry implements BaseColumns {

        public static final String TABLE_NAME = "groups";

        public static final String COLUMN_GROUP_NAME = "groupName";
    }
}

File: mobile/src/main/kotlin/com/planit/mobile/data/Contracts/GroupContract.kt
package com.planit.mobile.data.Contracts

import android.provider.BaseColumns

/**
 * Created by Home on 2017-08-02.
 */

class GroupContract {

    class GroupEntry : BaseColumns {
        companion object {

            val _ID = "_id"

            val TABLE_NAME = "groups"

            val COLUMN_GROUP_NAME = "groupName"
        }
    }
}
